odgovlt.py

```diff
@@ -73,6 +73,9 @@ def import_stage(self, harvest_object):
         base_context = {'model': model, 'session': model.Session,
                         'user': self._get_user_name()}
         data_to_import = json.loads(harvest_object.content)
+        R_ZODZIAI_STRING = '%s' % data_to_import['R_ZODZIAI']
+        R_ZODZIAI_STRING = R_ZODZIAI_STRING.replace(u'\u200b', '')
+        R_ZODZIAI = R_ZODZIAI_STRING.split(', ')
```

Comment:
All capital case variables should be used only for constants, and these variables does not look like constats. Please fallow conventions described in PEP-8 [1].

[1] https://www.python.org/dev/peps/pep-0008/#naming-conventions

Also, do not try to reinvent the wheel. Most of functionality for transfering data from opendata.gov.lt to CKAN was already implemented. And there is no point for you to reimplement everything again. Try to reuse as much as possible from existing implementatino:

https://github.com/ivpk/opendata.gov.lt-mysql-import/tree/3c22ae02e2b7114d1cb8e66eaed6de4d09f529fa

For example, case with keywords is not that simple as you think. Keywords can be split using `,` and `;` symbols. You can find previous emplementatino here:

https://github.com/ivpk/opendata.gov.lt-mysql-import/blob/3c22ae02e2b7114d1cb8e66eaed6de4d09f529fa/odgovltimport.py#L112-L123